frontend/src/plugins/vuetify.js

```diff
@@ -6,6 +6,7 @@ import PbsLogo from '@/assets/PbsLogo.svg'
 import GoogleLogo from '@/assets/GoogleLogo.svg'
 import eCampLogo from '@/assets/eCampLogo.svg'
 import CeviLogo from '@/assets/CeviLogo.svg'
+import JublaLogo from '@/assets/JublaLogo.svg'
```

Comment:
I used https://cdn.jubla.ch/target/c9b79b916f2f4a68676a03b7156c09af36999c49/jubla-logo%20(1).svg which is extracted from the website https://www.jubla.ch. However the original file contains a `style` tag that vuetify doesn't seem to handle correctly (returns an empty svg definition). I removed it for now. @manuelmeister any idea how to fix this?

Reply:
Not on my machine:
![Screenshot 2022-12-06 at 12-47-28 eCamp3](https://user-images.githubusercontent.com/7566995/205903597-033f697a-e86b-4b36-879c-83352af0d16d.png)

Reply:
![Bildschirmfoto 2022-12-06 um 14 51 36](https://user-images.githubusercontent.com/3001985/205930164-b9158caf-d146-4cf0-ac30-2b2be71816e8.png)

Reply:
Thx @manuelmeister !